coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -7,7 +7,7 @@ set oem_id="ec2"
 set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0"
 
 if [ "$grub_platform" = pc ]; then
-	set linux_console="console=ttyS0,115200n8"
+	set linux_console="console=ttyS0,115200n8 nvme_core.io_timeout=4294967295 nvme_core.max_retries=10"
```

Comment:
this should be on the `linux_append` line since it's independent of bios booting (`pc` in grub terms) or uefi booting.